### DISCUSSION TITLE
Changing region for firebase hosting

### DISCUSSION THREAD

**Discussion by adityar15** [nitrojs/nitro#637] [Q&A]
Whenever we create a nitro preset with firebase for the SSR Nuxt 3 app, the default region of the function is us-central-1. Is it possible to change this region?

Thanks :)

**Comment by Sandlens** [comment]
~~I think you can set the `region` under `rewrites` in `firebase.json` as per the [Firebase documentation](https://firebase.google.com/docs/hosting/full-config#rewrite-functions).~~

EDIT: I tried this and it doesn't work. Tracking here: #90

**Reply by husayt** [reply]
see [here](https://nitro.unjs.io/deploy/providers/firebase#options)

**Reply by husayt** [reply]
it does change it for deploy, but for emulator it is still going for default region
